daprdocs/content/en/developing-applications/local-development/sdk-serialization.md

```diff
@@ -109,7 +100,9 @@ object before saving it.
     await client.PublishEventAsync("MyPubSubName", "TopicName", "My Message");
 ```
 
-The event is published and the content is serialized to `byte[]` and sent to Dapr sidecar. The subscriber receives it as a [CloudEvent](https://github.com/cloudevents/spec). Cloud event defines `data` as String. The Dapr SDK also provides a built-in deserializer for `CloudEvent` object. 
+The event is published and the content is serialized to `byte[]` and sent to Dapr sidecar. The subscriber receives it 
```

Comment:
Not sure what has changed here, just looks like formatting. Can you undo this change?

Reply:
Done